src/api/entities.js

```diff
@@ -112,14 +112,14 @@ export function doUpdateSource(source, formData) {
             .then((authenticationDataOut) => {
                 return authenticationDataOut;
             }, (_error) => {
-                throw { error: 'Authentication update failure.' };
+                throw { error: { title: 'Authentication update failure.' } };
```

Comment:
Its not very good idea to completly discard the error response. If you dont want to show it as notification a least display it in console.

Reply:
This was just to preserve compatibility with new notification settings, the whole edit form should be changed at some time in the future :skull_and_crossbones: but I changed it